scripts/browserstack.py

```diff
@@ -40,15 +40,15 @@ def buildTestCommand(appToken, testToken):
     if "BASE_LOCATION" in os.environ:
         baseLoc = os.environ["BASE_LOCATION"]
 
-    releaseAppBundle = baseLoc + "bundle/commcareRelease/app-commcare-release.aab"
+    releaseApk = baseLoc + "apk/commcare/release/app-commcare-release.apk"
```

Comment:
can we define this switch as a Jenkins flag so that we don't need to make a code change in future for shifting across apk and bundle

Reply:
Yeah, we can have two variables there instead of `BASE_LOCATION` that gives the exact location of release apk and test apk.